libimage/copier.go

```diff
@@ -49,6 +49,10 @@ type CopyOptions struct {
 	CompressionFormat *compression.Algorithm
 	// CompressionLevel specifies what compression level is used
 	CompressionLevel *int
+	// ForceCompressionFormat ensures that the compression algorithm set in
+	// DestinationCtx.CompressionFormat is used exclusively, and blobs of other
```

Comment:
I don't think it's clear what `DestinationCtx.CompressionFormat` is in `libimage`.

I think it's important to highlight why setting `CompressionFormat` may not suffice.

Reply:
It should be `CompressionFormat` I have amended this.